data/loftwah.json

```diff
@@ -202,6 +207,14 @@
       "color": "#EF5904",
       "description": "I accepted the role as EddieHub Ambassador 0.2",
       "url": "https://www.eddiehub.org/ambassador-program"
+    },
+    {
+      "title": "Received my first GitHub Sponer",
```

Comment:
Just wondered if you meant `Sponsor` rather than Sponer

Reply:
Hahaha great pickup

Reply:
I didn't want to assume it wasn't an Australian term... 😜

Reply:
No, I'm just a little bit special 😅 I have fixed it. No more GitHub Sponers for me.